Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/WheelWithBypassDamper.mo

```diff
@@ -0,0 +1,296 @@
+    final VSup_flow_nominal=m1_flow_nominal/1.293)
+    "Calculates the effectiveness of heat exchange"
+    annotation (Placement(transformation(extent={{-100,0},{-80,20}})));
+  Buildings.Fluid.Sensors.VolumeFlowRate VExh_flow(
+   redeclare package Medium = Medium2, final m_flow_nominal=m2_flow_nominal)
+    "Flow sensor in the exhaust air stream"
+    annotation (Placement(transformation(extent={{-42,-28},{-56,-12}})));
+  Buildings.Fluid.Sensors.VolumeFlowRate VSup_flow(
+   redeclare package Medium = Medium1, final m_flow_nominal=m1_flow_nominal)
+    "Flow sensor in the supply air stream"
+    annotation (Placement(transformation(extent={{-78,28},{-64,44}})));
+  Buildings.Fluid.Actuators.Dampers.Exponential bypDamSup(
+    redeclare package Medium = Medium1,
+    final m_flow_nominal=m1_flow_nominal,
+    final dpDamper_nominal=dp1_nominal)
+    "Bypass damper in the supply air stream"
+    annotation (Placement(transformation(extent={{-38,70},{-18,90}})));
+  Buildings.Fluid.Actuators.Dampers.Exponential damSup(
+    redeclare package Medium = Medium1,
+    final m_flow_nominal=m1_flow_nominal,
+    final dpDamper_nominal=dp1_nominal)
+    "Damper in the supply air stream"
+    annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=0,
+        origin={-32,36})));
+  Buildings.Fluid.Actuators.Dampers.Exponential damExh(
+    redeclare package Medium = Medium2,
+    final m_flow_nominal=m2_flow_nominal,
+    final dpDamper_nominal=dp2_nominal)
+    "Damper in the exhaust air stream"
+    annotation (Placement(transformation(
+        extent={{10,10},{-10,-10}},
+        rotation=-90,
+        origin={60,-40})));
+  Buildings.Fluid.Actuators.Dampers.Exponential bypDamExh(
+    redeclare package Medium = Medium2,
+    final m_flow_nominal=m2_flow_nominal,
+    final dpDamper_nominal=dp2_nominal)
+    "Bypass damper in the exhaust air stream"
+    annotation (Placement(transformation(extent={{-20,-70},{-40,-50}})));
+  Modelica.Blocks.Sources.RealExpression TSup(
+      final y=Medium1.temperature(
+      Medium1.setState_phX(
+        p=port_a1.p,
+        h=inStream(port_a1.h_outflow),
+        X=inStream(port_a1.Xi_outflow))))
+    "Temperature of the supply air"
+    annotation (Placement(transformation(extent={{-160,-30},{-140,-10}})));
+  Modelica.Blocks.Sources.RealExpression TExh(
+      final y=Medium2.temperature(
+      Medium2.setState_phX(
+        p=port_a2.p,
+        h=inStream(port_a2.h_outflow),
+        X=inStream(port_a2.Xi_outflow))))
+    "Temperature of the exhaust air"
+    annotation (Placement(transformation(extent={{-160,-50},{-140,-30}})));
+  Modelica.Blocks.Sources.RealExpression PEle(y=P_nominal)
+    "Calculates the electric power consumption"
+    annotation (Placement(transformation(extent={{70,-10},{90,10}})));
+  Modelica.Fluid.Interfaces.FluidPort_a port_a1(
+    redeclare final package Medium = Medium1)
+    "Fluid connector a1 of the supply air"
```

Comment:
update the comments to be like `Fluid connector a1 of the supply air (positive design flow direction is from port_a1 to port_b1)`